src/CommandingOfficers/TrilogyCommander.java

```diff
@@ -0,0 +1,64 @@
+package CommandingOfficers;
+
+import Engine.GameScenario;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitModifier;
+import Terrain.MapMaster;
+import Units.Unit;
+import Units.UnitDelta;
+import java.util.ArrayList;
+import CommandingOfficers.CommanderInfo.InfoPage;
+
+/** Base class for AW2/3 COs (AW1 is special) */
+public abstract class TrilogyCommander extends Commander
```

Comment:
Is `TrilogyCommander` appropos if it only represents 2&3? Just splitting hairs here ofc.

Reply:
Do you want to read `AWTwoThreeCommander`?

Reply:
I want to read _accurate_ things, of course.